snake_n_ladders/lib/game_configuration.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class GameConfiguration
+  module EndGameRule
+    TILL_LAST_PLAYER = :till_last_player
+    FIRST_TO_FINISH = :first_to_finish
+
+    def self.all
+      [TILL_LAST_PLAYER, FIRST_TO_FINISH]
+    end
+  end
+
+  module ExtraDiceRoll
+    ON_LENGTH = :on_length
+    NONE = :none
+
+    def self.all
+      [ON_LENGTH, NONE]
+    end
+  end
+
+  module InvalidDiceRoll
+    THREE_SIXES = :three_sixes
+    NONE = :none
+
+    def self.all
+      [THREE_SIXES, NONE]
+    end
+  end
```

Comment:
shouldn't this should inherit from StandardError class?

Reply:
nice suggestion, will incorporate this